jobs/resource_job_test.go

```diff
@@ -1973,7 +1975,7 @@ func TestResourceJobCreateFromGitSourceWithoutProviderFail(t *testing.T) {
 			}
 		}
 	`,
-	}.ExpectError(t, "git source is not empty but Git Provider is not specified and cannot be guessed by url &{Url:https://custom.git.hosting.com/databricks/terraform-provider-databricks Provider: Branch: Tag:0.4.8 Commit: JobSource:<nil>}")
```

Comment:
this test should not be changed

Reply:
I see, yeah if we are inferring the value of provider then I agree this shouldn't be changed. Let me update it